src/Symfony/Bridge/Twig/Extension/DumpExtension.php

```diff
@@ -23,10 +24,12 @@
 class DumpExtension extends \Twig_Extension
 {
     private $cloner;
+    private $dumper;
 
-    public function __construct(ClonerInterface $cloner)
+    public function __construct(ClonerInterface $cloner, DataDumperInterface $dumper = null)
```

Comment:
How would the same dumper object be used each time, but separate output returned from `DumpExtension::dump()`?
Also since that method is called from twig, how does that dumper argument get passed in?